ftf_cli/commands/add_input.py

```
@@ -177,9 +186,8 @@ def add_input(path, profile, name, display_name, description, output_type):
 
         click.echo(f"✅ Input added to the {facets_yaml}.")
 
-    except Exception:
-        traceback.print_exc()
-        raise click.UsageError(f"❌ Error encountered while adding input {name}")
+    except Exception as e:
+        raise click.UsageError(f"❌ Error encountered while adding input: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)